packages/jellyfin-server/build.sh

```diff
@@ -0,0 +1,267 @@
+TERMUX_PKG_HOMEPAGE="https://jellyfin.org"
+TERMUX_PKG_DESCRIPTION="A free media system for organizing and streaming media (server)"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="10.10.7"
+_FFMPEG_VERSION="7.0.2-9"
+TERMUX_PKG_SRCURL=(
+	"https://github.com/jellyfin/jellyfin/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+	"https://github.com/jellyfin/jellyfin-ffmpeg/archive/refs/tags/v${_FFMPEG_VERSION}.tar.gz"
+)
+_GIT_URL=(
+	"git+https://github.com/ericsink/cb.git"
+	"git+https://github.com/ericsink/SQLitePCL.raw.git"
+	"git+https://github.com/mono/skia.git"
+	"git+https://github.com/jellyfin/jellyfin-web.git"
+)
+_GIT_COMMIT=(
+	cd2922b8867e4360f0976601414bd24a3ad613d8
+	7521c274efb2b49855b192f17862e87964460bac
+	4bed689c9c9eb77a120c6a9d54af6a572c85d1c2
+	f4b8aa0ed4c5b571a3a6cc3bb027bb8ecebe5b68
+)
+_GIT_BRANCH=(
+	master
+	no-xamarin
+	release/2.x
+	v10.10.7
+)
+_GIT_SHA256=(
+	1107df127ead66ace2baf92467fa10a8215371741e0b0b6a0580496ff1ae0bcf
+	8cd1b773026da818c47c693cf7a4bb81ab927b869a0ce2c6abcdbbe38d79922b
+	46c733f05df257e4bec13c09b53e7ef39bb80f21acb435d206dce609f4175ab1
+	2b8b180bdba21acc574858e50eded5fa1f7575f2571a8f17a9da7e2d9a79bb66
+)
+TERMUX_PKG_SHA256=(
+	797db59e50e33ecf85562f6c49651963bd5f00dd9cb74bf89dd905513c8207ec
+	ae4ea57516e606a73fd2745b21284c65d41d3851d05a2ac17c425d7488192ba0
+)
+TERMUX_PKG_SUGGESTS="jellyfin-ffmpeg"
+TERMUX_PKG_BUILD_DEPENDS="aspnetcore-targeting-pack-8.0, dotnet-targeting-pack-8.0, libcairo, pango, libjpeg-turbo, giflib, librsvg"
+TERMUX_PKG_DEPENDS="libc++, fontconfig, aspnetcore-runtime-8.0, dotnet-host, dotnet-runtime-8.0, sqlite, libexpat, libpng, libwebp, freetype, ffmpeg"
+TERMUX_PKG_SERVICE_SCRIPT=(
+	"jellyfin"
+	"exec ${TERMUX_PREFIX}/bin/jellyfin 2>&1"
+)
+TERMUX_PKG_EXCLUDED_ARCHES="arm"
+TERMUX_PKG_RM_AFTER_INSTALL="
+opt/jellyfin/include
+opt/jellyfin/lib/pkgconfig
+opt/jellyfin/share
+"
+termux_step_post_get_source() {
+	# this portion is required until TERMUX_PKG_SRCURL adds support for multiple git repos
+	local _idx;
+	local _sha256sums=();
+	local _error=0;
+	( [ "${#_GIT_URL[@]}" -eq "${#_GIT_COMMIT[@]}" ] && [ "${#_GIT_URL[@]}" -eq "${#_GIT_BRANCH[@]}" ] ) || termux_error_exit '_GIT_URL, _GIT_COMMIT and _GIT_BRANCH must be arrays of the same length'
```

Comment:
Our build scripts are written for Bash, and arithmetic evaluation lets this be expressed a bit more nicely.
```suggestion
	(( ${#_GIT_URL[@]} == ${#_GIT_COMMIT[@]} && ${#_GIT_URL[@]} == ${#_GIT_BRANCH[@]} )) || termux_error_exit '_GIT_URL, _GIT_COMMIT and _GIT_BRANCH must be arrays of the same length'
```